specification/gp-connect-access-record-fhir.yaml

```
@@ -33,184 +40,264 @@ info:
+    The GP Connect Access Record: Structured generic hazard log shows you the clinical risks of using GP Connect functionality through a consumer system. Use it to identify the clinical risks of consuming clinical data and presenting it in your system, and record these in your own system-specific hazard log. If necessary, take mitigating action to ensure clinical safety.
 
-    [Clinical safety principles](https://github.com/nhsconnect/gpc-consumer-support/wiki/Clinical-Safety-Principles) shows you who is responsible for the clinical safety of GP Connect specifications and consumer systems.
+    ## Errors
 
+    We use standard HTTP status codes to show whether an API request succeeded or not. They are usually in the range:
 
+    - 200 to 299 if it succeeded, including code 202 if it was accepted by an API that needs to wait for further action
 
-    ## Hazard log
+    - 400 to 499 if it failed because of a client error by your application
 
-    The GP Connect Access Record: Structured generic hazard log shows you the clinical risks of using GP Connect functionality through a consumer system. Use it to identify the clinical risks of consuming clinical data and presenting it in your system, and record these in your own system-specific hazard log. If necessary, take mitigating action to ensure clinical safety.
+    - 500 to 599 if it failed because of an error on our server
 
+    Errors specific to each API are shown in the Endpoints section, under Response. See our [reference guide](https://digital.nhs.uk/developer/guides-and-documentation/reference-guide#http-status-codes) for more on errors.
   contact:
     name: 'gp-connect-access-record-fhir API Support'
     url: 'https://digital.nhs.uk/developer/help-and-support'
     email: api.management@nhs.net
 servers:
-  - url: 'https://sandbox.api.service.nhs.uk/gp-connect-access-record-fhir'
+  - url: 'https://sandbox.api.service.nhs.uk/gp-connect/patient-facing/get-structured-record'
     description: Sandbox environment.
-  - url: 'https://int.api.service.nhs.uk/gp-connect-access-record-fhir'
+  - url: 'https://int.api.service.nhs.uk/gp-connect/patient-facing/get-structured-record'
     description: Integration test environment.
-  - url: 'https://api.service.nhs.uk/gp-connect-access-record-fhir'
+  - url: 'https://api.service.nhs.uk/gp-connect/patient-facing/get-structured-record'
     description: Production environment.
-paths: {}
+paths: 
+  /FHIR/STU3/Patient/$access-record-structured:
```

This is the same operation as access record structured in GPC - if it's the same operation then the operation name needs to be consistent.

```suggestion
  /FHIR/STU3/Patient/$gpc.getstructuredrecord
```